pyproject.toml

```diff
@@ -152,25 +152,6 @@ version_files = [
     'docs/package.json:^  "version":',
 ]
 
-[tool.commitizen.customize]
-commit_parser = '^(?P<change_type>feat|fix|refactor|perf|break|docs)(?:\((?P<scope>[^()\r\n]*)\)|\()?(?P<breaking>!)?:\s(?P<message>.*)?'
-change_type_order = [
-  "BREAKING CHANGES",
-  "✨ New",
-  "🐛 Fixes",
-  "⚙️ Under the Hood",
-  "⚡ Performance Improvements",
-  "📚 Documentation Improvements",
-]
-
-[tool.commitizen.customize.change_type_map]
-break = "BREAKING CHANGES"
-feat = "✨ New"
-fix = "🐛 Fixes"
-refactor = "⚙️ Under the Hood"
-docs = "📚 Documentation Improvements"
-perf = "⚡ Performance Improvements"
-
```

Comment:
@WillDaSilva - For my info, how does this play with the changelog automation? Is this mapping moving to another file, for instance?

Reply:
It has been moved to https://github.com/meltano/commitizen-version-bump/blob/main/cz_version_bump/__init__.py

Reply:
Because of this, it applies to both Meltano and the SDK, and any other repository we want to version in the future (e.g. the EDK).

Reply:
@WillDaSilva - Thanks! Sorry, I totally missed that this was a new custom repo with our logic baked in. Very cool!